src/Between.php

```diff
@@ -164,6 +164,10 @@ public function isValid($value)
     {
         $this->setValue($value);
 
+        if ($this->getMin() === null || $this->getMax() === null) {
+            throw new Exception\InvalidArgumentException("Missing option. 'min' and 'max' have to be given");
```

Comment:
I disagree with this approach. What's the actual use-case for this change?

I'm basically just missing this bit: why do you want to instantiate a validator and keep it in an invalid state?